### ISSUE TITLE
Consolidate DTS repos into atd-data-tech (monorepo)

### ISSUE THREAD

**Issue by amenity** [cityofaustin/atd-data-tech#1037]
Per #979, migrate all issues, labels, and templates into ATD Data Tech. 

## For each issue, we will:

1. Create an identical issue in atd-data-tech (same description, title, comments, pipeline, milestone, estimate)
2. Apply the correct `App` label to the new issue
3. Apply a special `DTS Migration` label to the new issue  for auditing purposes
4. Update the new issue's description with a link to the original issue.
4. Update the original issue with a link to the new issue.
5. Apply a special `Migrated` label to the original issue for auditing purposes

## Questions
- can epics be preserved? need to research if this is technically possible.
- i suppose we should timestamp the comments in the body of the comment, otherwise that information will be lost
- are we migrating closed issues? i think we should
- do we need to migrate comments?

**Comment by johnclary** [comment]
working from here:
https://github.com/cityofaustin/atd-monorepo-migration


We’re migrating all *OPEN* issues from all of our atd-… repos to the atd-data-tech repo
Once an issue is migrated, the original issue will be closed, the sprint will be removed, it will be tagged with a migrated label, and the issue description will be updated with a link to the new issue.
Assignments, milestones, epics, estimates, pipelines, dependencies, and releases will all be preserved :sparkles: with one important exception:
We’re not migrating closed issues. These will be dropped from epics, dependencies, and releases. Now is your chance to protest that decision.
Comments will be migrated, and a note will be prepended to each comment with the author’s username and date
Pull Requests, code, branches, etc will not be migrated. If you have an open PR connected to an open issue, you’ll need to re-link that PR to the new migrated issue in atd-data-tech.
New Product labels will be applied to issues with Project tags that pertain to an existing product (e.g. Vision Zero Viewer), or if the issue comes from a product-specific repo (e.g., atd-knack-data-tracker). Please use the Product labels going forward, and reach out to the PMs if you need clarification. They will be essential to filtering issues in lieu of separate dedicated repos
@amenity @jace we’ll need to tidy up the product labels post-migration, as there may be some redundant labels (e.g VZA App and Vision Zero in Action)
Issues migrated from the atd-geospatial repo will have the Service: Geo tag applied (most already have this tag :slightly_smiling_face: ).
Issue templates will not be migrated. You’ll have to rebuild these or copy/paste them from the .github folder of the old repo. While you’re at it, please add a Product label to you’re templates if appropriate.
The atd-service-bot will be updated to send all issues to the add-data-tech repo with the appropriate tags
Once the migration is complete, you should only create issues in the atd-data-tech repo. 
Brace yourself for an onslaught of GitHub notifications :grimacing: 

You can view the project & repos > product label mappings here:
https://docs.google.com/spreadsheets/d/1pTum5oFfDpIo575zAYDPA_bdsEfoElpCO1-HzAU68wk/edit#gid=651251559

whew, that’s a lot. send me your questions/concerns :sweat_smile: :pray:
